soh/soh/OTRGlobals.h

```diff
@@ -95,13 +98,14 @@ u8 Randomizer_GetSettingValue(RandomizerSettingKey randoSettingKey);
 RandomizerCheck Randomizer_GetCheckFromActor(s16 actorId, s16 actorParams, s16 sceneNum);
 void Randomizer_LoadHintLocations(const char* spoilerFileName);
 void Randomizer_LoadItemLocations(const char* spoilerFileName, bool silent);
-s16 Randomizer_GetItemModelFromId(s16 itemId);
-s32 Randomizer_GetItemIDFromGetItemID(s32 getItemId);
-s32 Randomizer_GetRandomizedItemId(GetItemID ogId, s16 actorId, s16 actorParams, s16 sceneNum);
-s32 Randomizer_GetItemIdFromKnownCheck(RandomizerCheck randomizerCheck, GetItemID ogId);
+s32 Randomizer_GetItemIDFromGetItemID(s32 getItemId); // TODO: Remove me
```

Comment:
todo?

Reply:
Ah, forgot about that one, need to double check if that's still used anywhere.

Reply:
Still being used in `z_sram.c`, but I think I can actually make it so it's not used anymore.

Reply:
Resolved by e1b403a31b1dde6ae190a2a8352d1f0fd69073dd